src/services/__tests__/profileService.test.ts

```diff
@@ -306,21 +354,20 @@ describe("ProfileService#updateProfile", () => {
   });
 
   it("update an user profile to the API", async () => {
-    mockUpdateProfile.mockImplementation(() =>
-      t.success(validApiProfileResponse)
-    );
+    mockUpdateProfile.mockImplementation(() => t.success(validApiProfileResponse));
 
     const service = new ProfileService(api);
 
     const res = await service.updateProfile(mockedUser, updateProfileRequest);
 
     expect(mockUpdateProfile).toHaveBeenCalledWith({
       fiscal_code: mockedUser.fiscal_code,
-      body: updateProfileRequest
+      body: { ...updateProfileRequest }
```

Comment:
Why this change? the previous version should work fine.

Reply:
right, it was a refactor zombie :) fixed in https://github.com/pagopa/io-backend/pull/930/commits/bca9af3338cc26e0c6145c308da15a736fa9cf14